build/Linux+BSD/portable/x86_64/Recipe

```diff
@@ -48,6 +48,7 @@ yum -y install \
   libxslt-devel \
   mesa-libGL-devel \
   portaudio-devel \
+  portmidi-devel \
```

Comment:
note: I would think the docker image build will need to be rerun, in order to get this additional library, so that it will be available to be included when building and packing.